src/components/Comments.jsx

```diff
@@ -0,0 +1,40 @@
+import {useEffect, useState} from 'react';
+import '../App.css';
+import { getCommentsByArticleIdApi } from '../utils/api';
+
+const Comments =({singleArticleID}) => {
+  
+  const [comments, setComments] = useState([]);
+  
+
+  useEffect(() => {
+    if (singleArticleID) {
+    getCommentsByArticleIdApi(singleArticleID).then((CommentsFromArticleById) => {     
```

Comment:
Avoid naming variables with capital letters please, as that tends to indicate classes, or components 🙂

Reply:
Ok will do.